src/providers/xref.provider.ts

```diff
@@ -30,14 +30,17 @@ function shouldProvide (context: Context): boolean {
 }
 
 async function getLabels (): Promise<string[]> {
-  const regex = /\\[\\[(\\w+)\\]\\]/g
-  const labels = await vscode.workspace.findFiles('**/*.adoc').then((files) =>
-    files
+  const regex = /\[\[(\w+)\]\]/g
+  const labels = await vscode.workspace.findFiles('**/*.adoc').then((files) => {
+    const contentOfFilesConcatenated = files
       .map((uri) => readFileSync(uri.path).toString('utf-8'))
```

Comment:
Not related but I think we should use https://code.visualstudio.com/api/references/vscode-api#FileSystem to `readFile`.

As far as I know, `readFileSync` won't be available/won't work in VS Code Web.

Reply:
agree.
I only reenabled what was already implemented.
I can change this Pull Request or update in another iteration.

(For information, planned to work on it on Friday 25th October)

Reply:
> I can change this Pull Request or update in another iteration.

I'm fine with both!

Reply:
I would prefer to it in a seprate Pull request. I listed the task in https://github.com/asciidoctor/asciidoctor-vscode/issues/648#issuecomment-1327445736